packages/cpp/src/browser/cpp-commands.ts

```diff
@@ -11,41 +11,81 @@ import { CommandContribution, CommandRegistry, Command } from '@theia/core/lib/c
 import URI from "@theia/core/lib/common/uri";
 import { open, OpenerService } from '@theia/core/lib/browser';
 import { CppClientContribution } from "./cpp-client-contribution";
-import { TextDocumentItemRequest } from "./cpp-protocol";
+import { TextDocumentItemRequest, AnyRequest } from "./cpp-protocol";
 import { TextDocumentIdentifier } from "@theia/languages/lib/common";
+import { DirNode, FileDialogFactory, FileStatNode } from '@theia/filesystem/lib/browser';
 import { EditorManager } from "@theia/editor/lib/browser";
+import { FileSystem } from '@theia/filesystem/lib/common';
+import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
 
-/**
- * Switch between source/header file
- */
-export const SWITCH_SOURCE_HEADER: Command = {
-    id: 'switch_source_header',
-    label: 'Switch between source/header file'
-};
+export namespace Commands {
+
+    export const OPEN: Command = {
+        id: 'cpp:open',
+        label: 'Specify folder location...'
```

Comment:
It is possible to add a more descriptive title, even if a bit ugly? Like "Select folder containing compile_commands.json"